libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -523,6 +519,32 @@ impl NodeAPI for Greenlight {
         Ok(client.withdraw(request).await?.into_inner())
     }
 
+    async fn prepare_withdraw(
+        &self,
+        to_address: String,
+        fee_rate_sats_per_vbyte: u32,
+    ) -> Result<TxprepareResponse> {
```

Comment:
interface looks good here and also in breez_services.rs.
The return value should be an sdk struct defined in models.rs taht contains the raw tx and the relative + absolute fee info. Something as follows:

```
pub struct PrepareWithdrawResponse {
  pub raw_tx_hex: String,
  pub sat_per_vbyte: u32,
  pub fee_sat: u64
}
```

The looks good IMO also as a return result from breez_services.